src/main/java/View/Configuration/ThymeleafConfig.java

```diff
@@ -15,10 +15,15 @@
 public class ThymeleafConfig implements ServletContextListener {
 
     public void contextInitialized(ServletContextEvent sce) {
-        TemplateEngine engine = templateEngine(sce.getServletContext());
-        ContextUtil.storeTemplateEngine(sce.getServletContext(), engine);
+        ServletContext sceInternal = sce.getServletContext();
+        sceInternal.setRequestCharacterEncoding("UTF-8");
```

Comment:
setRequestCharacterEncoding() si vymazal z toho povodneho miesta kde bolo? lebo to tu nevidim cervenym

Reply:
Práveže som ho hľadal a nikde nenašiel tiež tomu nerozumiem, tak som to sem pridal.

Reply:
doteraz sa to volalo v super.doGet t.j. ConnectionServlet